packages/pds-ember/app/styles/pds/core/typography/base.scss

```diff
@@ -0,0 +1,11 @@
+@use "config" as *;
+
+@use "../../theme";
+
+html {
+  font-family: theme.$fontFamily;
```

Comment:
This may be migrated to `<Pds::App>`, but we'll evaluate the possibility when we get around to adding that Ember component.